aea/cli/common.py

```diff
@@ -287,7 +288,7 @@ class AEAConfigException(Exception):
 class ConnectionsOption(click.Option):
```

Comment:
here we implement the conversion of public id list in the `aea run --connections pid_list` command.

E.g. now the input must be something like:
`aea run --connections "fetchai/oef:0.1.0, fetchai/local:0.1.0"`

Reply:
Cool!